sys/kern/bus.c

```diff
@@ -87,6 +87,8 @@ void generic_bs_write_region_4(bus_space_handle_t handle, bus_size_t offset,
     *dst++ = *src++;
 }
 
+#define DEV_IDENTIFIED_BY_PARENT ((void *)0xffffffff)
```

Comment:
Any reason for this address? I'm afraid this could easily cause an error.
Why not `NULL`?

Reply:
It cannot be ```NULL``` without further changes to ```bus_generic_probe```. In ```bus_generic_probe```there is this code:
```
if (dev == NULL)
      continue;
if (dev == DEV_IDENTIFIED_BY_PARENT) {...}
``` 
I suppose the reason for this address being ```0xffffffff``` is that without changes in the logic of  ```bus_generic_probe``` this value cannot be ```NULL```, and there can't be a device at ```0xffffffff```.

Reply:
In that case in my opinion `-1` is better -- more portable -- value. Please remember that we support _AArch64_ where pointers have 64 bits.